.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Any reason this can't be just

```yaml
on: push
```

Reply:
It's not happening, because the workflow file is not in the repo yet. If both `push` and `pull_request` triggers are enabled, GH will run two actions for branches+PRs pushed by you. If that doesn't bother you, feel free to remove the branch filter.

Reply:
Ah, I see. OK, this way is better 👍